samples/tests/export_model_tables_classification_test.py

```diff
@@ -0,0 +1,43 @@
+# Generated code sample for google.cloud.aiplatform.DatasetServiceClient.get_dataset
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+import os
+
+from samples import export_model_tables_classification_sample
+from google.cloud import storage
+
+PROJECT_ID = os.getenv("BUILD_SPECIFIC_GCLOUD_PROJECT")
+MODEL_ID = "5359002081594179584"  # iris 1000
+GCS_URI = "gs://ucaip-samples-test-output/tmp/export_model_test"
```

Comment:
I thought I had a comment regarding this before as well, but may have missed it. 

Fixed resources should be passed in by an environment variables. See [External Resources](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#external-resources) section of the sample style guide. 

Users and googlers without access to these resources should be able to run tests on these samples w/o having to modify them

Reply:
I changed those two instances to `os.getenv("XYZ")`, but I don't think I'm resolving the root issue. From the style guide, resources

> should be identified and passed in through an environment variable

Does **identifying** == the ID of the resource in a comment, for the user to see? Thanks!

Reply:
No, you don't need to add a comment. The intended meaning is probably more literally "specify". 

Essentially what we want here is that I as a developer, should be able to run the tests on these samples without modifying them to confirm they still work. However unless I'm on the team maintaining the samples, I likely don't have access to the resources it's tested with. By using env variables means all I need to do when running locally is change the env var, and not worry about changing the model ids when there is a problem. 

So something like this is perfectly acceptable:
```python
resource_id = os.getenv("YOUR-RESOURCE-NAME")
```

For bonus points you can include a link to instructions for creating the resource, if you really want to make it accessible for other developers.

Reply:
TO-DO - replace hardcoded resource values with environment variables in all samples